agent/provider/tests/manifest_negotiator.rs

```diff
@@ -304,35 +268,29 @@ fn manifest_negotiator_test_with_node_identity(
 }
 
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "all", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "all"}}"#, // rulestore config
     r#"["https://domain.com"]"#, // compManifest.net.inet.out.urls
     None; // error msg
     "Accepted because everyone is set to all"
 )]
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "whitelist"}}"#, // rulestore config
     r#"["https://domain.com"]"#, // compManifest.net.inet.out.urls
     None; // error msg
     "Accepted because everyone whitelist is matching"
 )]
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "whitelist", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "whitelist"}}"#, // rulestore config
     r#"["https://non-whitelisted.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule: Invalid signature"); // error msg
+    Some("Everyone rule didn't match whitelist ;"); // error msg
     "Rejected because everyone whitelist mismatched"
 )]
 #[test_case(
-    r#"{"outbound": {"enabled": true, "everyone": "none", "audited-payload": {"default": {"mode": "all", "description": ""}}}}"#, // rulestore config
+    r#"{"outbound": {"enabled": true, "everyone": "none"}}"#, // rulestore config
     r#"["https://domain.com"]"#, // compManifest.net.inet.out.urls
-    Some("Audited-Payload rule: Invalid signature"); // error msg
+    Some("Everyone rule is disabled ; Audited-Payload rule: Invalid signature ; Partner rule requires node descriptor ;"); // error msg
```

Comment:
Audited payload message is sufficient. This group of tests focus on Audited-Payload logic only.

Reply:
Done